pkg/services/resourcesFilterService.go

```diff
@@ -200,5 +217,19 @@ func doesRegexMatchString(regex string, str string) bool {
 }
 
 func isOpenshiftResourceThatShouldBeEvaluated(managedFields []ManagedFields) bool {
-	return doesAtLeastOneFieldManagerStartWithOneOfThePrefixes(managedFields, []string{"openshift-controller-manager", "openshift-apiserver"}) || isAtLeastOneFieldManagerEqualToOneOfTheExpectedFieldManagers(managedFields, []string{"oc"})
+	openshiftManagedFields := []string{"openshift-controller-manager", "openshift-apiserver", "oc"}
```

Comment:
for "openshift-controller-manager" & "openshift-apiserver", this changes the behavior from "starts with" to "contains", why do we want this?